lib/segment/src/vector_storage/query_scorer/sparse_custom_query_scorer.rs

```diff
@@ -60,6 +60,7 @@ impl<TVectorStorage: SparseVectorStorage, TQuery: Query<SparseVector>> QueryScor
             .get_sparse(idx)
             .expect("Failed to get sparse vector");
 
+        // not exactly correct for Gridstore where the indices are compressed into u8
```

Comment:
Do we not care about the wrong measurements here or is the comment suggesting that we need to fix this in a separate PR?